_articles/it/building-community.md

```diff
@@ -0,0 +1,280 @@
+---
+lang: en
+title: Building Welcoming Communities
+description: Building a community that encourages people to use, contribute to, and evangelize your project.
+class: building
+toc:
+  setting-your-project-up-for-success: "Setting your project up for success"
+  growing-your-community: "Growing your community"
+  resolving-conflicts: "Resolving conflicts"
```

Comment:
The `toc` section should be removed as per point 5 of our [Contribution guidelines](https://github.com/github/opensource.guide/blob/e2c0ae140e50cfa83b8a5afeb4a6fd2c26d75910/docs/translations.md#starting-a-translation)